### ISSUE TITLE
Handling attribute names with `-`

### ISSUE THREAD

**Issue by dpc** [lfairy/maud#10]
The following fails:

```
button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-              
  target="#navbar" aria-expanded="false" aria-controls="navbar" {
    ...
}
```

How do I handle dash signs in attribute names?

**Comment by lfairy** [comment]
Hmm... currently Maud parses attribute names as identifier tokens, which falls flat when there are hyphens in the name.

I think we'll need a deeper refactoring to change that behavior. The parser's pretty hacky right now, and this is a good reason to start cleaning it up.

As for workarounds, I guess you can do this (using `$$` to disable escaping):

```rust
$$"<button type=\"button\" aria-expanded=\"false\">" {
   ...
} $$"</button>"
```

but of course that's not ideal.

**Comment by dpc** [comment]
Thanks for the workaround. That should keep me going.

**Comment by lfairy** [comment]
Fixed in 0.6.0. Thanks for the report!
